tensorflow/python/keras/layers/convolutional_test.py

```diff
@@ -336,6 +336,14 @@ def test_conv3dtranspose_constraints(self):
       self.assertEqual(layer.kernel.constraint, k_constraint)
       self.assertEqual(layer.bias.constraint, b_constraint)
 
+  def test_conv3dtranspose_dynamic_shape(self):
+    with self.session(use_gpu=True):
+      # Won't raise error here.
+      layer = keras.layers.Conv3DTranspose(3, 3, data_format='channels_last')
```

Comment:
This unit test tests nothing. Use `testing_utils.layer_test` instead.

Reply:
Fixed. Could you take another look?